CODEOWNERS

```diff
@@ -242,6 +242,9 @@
 /drivers/crypto/*nrf_ecb*                 @maciekfabia @anangl
 /drivers/display/display_framebuf.c       @dcpleung
 /drivers/dac/                             @martinjaeger
+/drivers/dai/                             @juimonen
```

Comment:
What about Arsen and  @dbaluta here too ?

Reply:
yes, please just suggest the correct persons here, I just silenced the CI with my own username